components/tooltip/test/tooltip-help.axe.js

```diff
@@ -0,0 +1,34 @@
+import '../tooltip.js';
+import '../tooltip-help.js';
+import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+
+const tooltipFixture = html`<d2l-tooltip-help text="Helpful label.">Contents should elaborate on the label (be short and concise)</d2l-tooltip-help>`;
+
+describe('d2l-tooltip-help', () => {
+
+	it('should pass all aXe tests (hide)', async() => {
+		const helpTooltip = await fixture(tooltipFixture);
+
+		await expect(helpTooltip).to.be.accessible();
+	});
+
+	it('should pass all aXe tests for state info (show)', async() => {
+		const helpTooltip = await fixture(tooltipFixture);
+		const actualTooltip = helpTooltip.shadowRoot.querySelector('d2l-tooltip');
+
+		actualTooltip.setAttribute('state', 'info');
+		actualTooltip.setAttribute('showing', 'showing');
```

Comment:
You may need an `await actualTooltip.updateComplete;` here... otherwise I'm not sure it's actually testing what you think it is.